libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/test/AddGranteeSelect.test.tsx

```diff
@@ -11,10 +11,12 @@ import { IWorkspaceUser, IWorkspaceUserGroup } from "@gooddata/sdk-backend-spi";
 import { groupAll, workSpaceGroup, workspaceUser } from "./GranteeMock";
 import { mapWorkspaceUserGroupToGrantee, mapWorkspaceUserToGrantee } from "../../shareDialogMappers";
 import { getGranteeItemTestId } from "../utils";
+import { uriRef } from "@gooddata/sdk-model";
 
 const defaultProps: IAddGranteeSelectProps = {
     onSelectGrantee: noop,
     appliedGrantees: [],
+    currentUserRef: uriRef(""),
```

Comment:
It would be better to have some actual string set, not just empty one. This looks weird.